Turn old admin interface into plugin

lkiesow opened the pull request:
This patch disables the old admin interface by default. Users can still activate this if they really want/need the old interface. This should force users to acknowledge and test the new interface while allowing a fallback in some edge cases.

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://docs.opencast.org/develop/developer/#participate/development-process/#automatically-closing-issues-when-a-pr-is-merged) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [ ] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

lkiesow committed:
Turn old admin interface into plugin

This patch disables the old admin interface by default. Users can still
activate this if they really want/need the old interface. This should
force users to acknowledge and test the new interface while allowing a
fallback in some edge cases.

gregorydlogan reviewed (commented):
This works, and looks good.  One minor nit-pick is the operations in `fast.xml` and `partial-preview.xml`, both of which contain a hardcoded `admin-ng` path component.  Not sure how to address that though.

lkiesow committed:
Update editor links

This patch updates the publication link for the internal publication to
point to the new editor.

lkiesow commented:
I've updated the workflows to point to the new editor.
Though the links aren't actually used anywhere in Opencast I think.
